AppCenterDistribute/AppCenterDistributeTests/Util/MSMockReachabilityUtil.m

```diff
@@ -0,0 +1,35 @@
+#import "MSMockReachabilityUtil.h"
+#import "MSTestFrameworks.h"
+
+static NSString *kMSNetworkReachabilityChangedNotificationName = @"kMSNetworkReachabilityChangedNotification";
+
+@interface MSMockReachabilityUtil ()
```

Comment:
You don't need "Util" here, `MSMockReachability` is enough. `MSMockKeychainUtil` contains "util" because the class it's mocking contains util but it's not the case here.

Reply:
changed